packages/opentelemetry-semantic-conventions/src/resource/SemanticResourceAttributes.ts

```diff
@@ -197,411 +879,968 @@ As an alternative, consider setting `faas.id` as a span attribute instead.
    */
-  SERVICE_NAMESPACE: 'service.namespace',
+  SERVICE_NAMESPACE: 'service.namespace';
 
   /**
    * The string ID of the service instance.
    *
    * Note: MUST be unique for each instance of the same `service.namespace,service.name` pair (in other words `service.namespace,service.name,service.instance.id` triplet MUST be globally unique). The ID helps to distinguish instances of the same service that exist at the same time (e.g. instances of a horizontally scaled service). It is preferable for the ID to be persistent and stay the same for the lifetime of the service instance, however it is acceptable that the ID is ephemeral and changes during important lifetime events for the service (e.g. service restarts). If the service has no inherent unique ID that can be used as the value of this attribute it is recommended to generate a random Version 1 or Version 4 RFC 4122 UUID (services aiming for reproducible UUIDs may also use Version 5, see RFC 4122 for more recommendations).
    */
-  SERVICE_INSTANCE_ID: 'service.instance.id',
+  SERVICE_INSTANCE_ID: 'service.instance.id';
 
   /**
    * The version string of the service API or implementation.
    */
-  SERVICE_VERSION: 'service.version',
+  SERVICE_VERSION: 'service.version';
 
   /**
    * The name of the telemetry SDK as defined above.
    */
-  TELEMETRY_SDK_NAME: 'telemetry.sdk.name',
+  TELEMETRY_SDK_NAME: 'telemetry.sdk.name';
 
   /**
    * The language of the telemetry SDK.
    */
-  TELEMETRY_SDK_LANGUAGE: 'telemetry.sdk.language',
+  TELEMETRY_SDK_LANGUAGE: 'telemetry.sdk.language';
 
   /**
    * The version string of the telemetry SDK.
    */
-  TELEMETRY_SDK_VERSION: 'telemetry.sdk.version',
+  TELEMETRY_SDK_VERSION: 'telemetry.sdk.version';
 
   /**
    * The version string of the auto instrumentation agent, if used.
    */
-  TELEMETRY_AUTO_VERSION: 'telemetry.auto.version',
+  TELEMETRY_AUTO_VERSION: 'telemetry.auto.version';
 
   /**
    * The name of the web engine.
    */
-  WEBENGINE_NAME: 'webengine.name',
+  WEBENGINE_NAME: 'webengine.name';
 
   /**
    * The version of the web engine.
    */
-  WEBENGINE_VERSION: 'webengine.version',
+  WEBENGINE_VERSION: 'webengine.version';
 
   /**
    * Additional description of the web engine (e.g. detailed version and edition information).
    */
-  WEBENGINE_DESCRIPTION: 'webengine.description',
+  WEBENGINE_DESCRIPTION: 'webengine.description';
 };
 
-export const CloudProviderValues = {
+/**
+ * Create exported Value Map for SemanticResourceAttributes values
+ * @deprecated Use the SEMRESATTRS_XXXXX constants rather than the SemanticResourceAttributes.XXXXX for bundle minification
+ */
+export const SemanticResourceAttributes: SemanticResourceAttributes =
```

Comment:
This doesn't need to redefine the documentation as it's referencing the interface which will provide it for this object.